src/style.css

```
@@ -1 +1,47 @@
-/* add styles here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1,
+.stars__star:nth-child(-n + 1) {
```

```suggestion
.stars__star:nth-child(-n + 1),
stars--2,
.stars__star:nth-child(-n + 2) ,
.... {
```
Consider grouping your selectors with commas